defaults/main.yml

```diff
@@ -117,7 +117,11 @@ neofs_storage__grpc_host_allow: []
 
 neofs_storage__control_enabled: False
 neofs_storage__control_authorized_keys: []
-neofs_storage__control_endpoint: 'localhost:8090'
+neofs_storage__control_endpoint_host: '{{ neofs_storage__bind_address }}'
```

Comment:
By default, control api endpoint should be bound to `localhost`.

Reply:
Fixed